wordbot.py

```diff
@@ -15,32 +16,40 @@ def lambda_handler(event, context):
     wordnikUrl = config['wordnik']['apiUrl']
     wordnikKey = config['wordnik']['apiKey']
     slackUrl = config['slack']['postUrl']
-    
+
     ### 1 - connect to WordNik API
     wclient = swagger.ApiClient(wordnikKey, wordnikUrl)
     wordsApi = WordsApi.WordsApi(wclient)
-    
+
     ### 2 - get the word of the day
     wod = wordsApi.getWordOfTheDay()
     if debug:
         print (wod.word)
         for definition in wod.definitions:
+            print (definition.partOfSpeech)
             print (definition.text)
 
     ### 3 - save word for later retrieval / comparison by the listener [TBD]
+    pass
 
-    ### 4 - post message to slack
-    msg = {}
-    
-    # this needs to be fleshed out a lot to have definitions, etc
-    # check slack docs on message formatting here:
-    # https://api.slack.com/docs/messages
-    msg['text'] = wod.word
+    ### 4 - format the message for posting
+    postText = "Hey, kids! Today's Secret Word is... %s\n\n" % wod.word.upper()
+    urlWord = wod.word.replace(' ', '%20');
+    postText += "*<http://wordnik.com/words/%s|%s>*\n" % (urlWord, wod.word)
+    for idx, definition in enumerate(wod.definitions):
```

Comment:
TIL the enumerate() function. I was planning to have to do manual index incrementation!